### ISSUE TITLE
freezing sapi 5 ESpeak

### ISSUE THREAD

**Issue by hozosch** [nvaccess/nvda#18760]
### Steps to reproduce:
Open a browser, such as Chrome, and when NVDA announces the window title, try stopping speech with the control key. It doesn't work.
### Actual behavior:-->
It takes a few seconds to respond
### Expected behavior:

it should respond immediatelly, like it did previously.
### NVDA logs, crash dumps and other attachments:
none
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
latest alpha and beta snapshots, it's very likely that changes to sapi 5 planned for 2025.3 caused this issue
#### Windows version:
windows 11 24h2 on ARM64
#### Name and version of other software in use when reproducing the issue:
Google chrome, file explorer, maybe any window when it gets the focus.
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Prior versions behave normally
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not an issue with com I believe, as other synths work as expected.

**Comment by dpy013** [comment]
hi @hozosch 
Can you make a screen recording or audio recording to reproduce the issue you mentioned?
thanks

**Comment by seanbudd** [comment]
This is probably a duplicate of #18674

does using [this snapshot build](https://github.com/nvaccess/nvda/actions/runs/16909830812/artifacts/3744991003) fix your problem?

**Comment by hozosch** [comment]
Yes, it does! Thanks!

**Comment by hozosch** [comment]
I noticed though that now you reduced the silence trimming, we have a bit of responsiveness sacrifice with ESpeak. I notice this in the log viewer in particular. I know it was too much for certain voices, is there a way to make this voice specific? I love how it is in 2025.1 and 2.
